nautobot/extras/views.py

```diff
@@ -2378,6 +2379,23 @@ class RelationshipListView(generic.ObjectListView):
 
 class RelationshipView(generic.ObjectView):
     queryset = Relationship.objects.all()
+    object_detail_content = ObjectDetailContent(
+        panels=(
+            ObjectFieldsPanel(
+                label="Relationship",
+                section=SectionChoices.LEFT_HALF,
+                weight=100,
+                fields=["label", "key", "type", "description", "required_on"],
+            ),
+            ObjectFieldsPanel(
+                label="Relationship Data",
```

Comment:
Maybe split this into "Source Attributes" and "Destination Attributes" panels? Might then need to have the main left panel use `exclude_fields` and have the two new right panels use `fields`?

Reply:
From latest commit:
![Screenshot 2025-01-30 at 5 25 10 PM](https://github.com/user-attachments/assets/160af4c4-ff25-4a1d-9e61-bab80bec0605)